app/javascript/components/widgets/numbers/falling-blocks.jsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { Engine, Render, World, Bodies, Sleeping, Composite } from 'matter-js';
```

Comment:
probably this should be squashed down into 67813e392d22077148d29b89ba87214e1192dae9, since that was the last reference to `Sleeping`